integration-tests/call-builder-return-value/lib.rs

```diff
@@ -148,7 +148,7 @@ mod call_builder {
             let selector = ink::selector_bytes!("get");
             let call = call_builder_call.delegate_call(code_hash, selector);
             let call_result = client
-                .call(&origin, &call, 0, None)
+                .call(&origin, &call, 0, None, None)
```

Comment:
Maybe now is the time to finally convert to a builder style API, since the default values `0, None, None` are the most common.

Reply:
Maybe it is 😄

Reply:
We should do a builder API, but maybe as a separate task where we can design it properly

I think a more minimal approach to this problem (applying the extra gas portion without breaking the `call` API) would be:

1. Have the `Client` store a value for the "extra portion"
2. Set it to some default low value e.g. 2%
3. Allow overriding the value via e.g. `client.set_extra_gas(2u8)` (this could also be configured via the e2e test attribute which initialises the client)